server/lib/transforms/link-analytics.js

```diff
@@ -1,13 +1,20 @@
 'use strict';
 
+const sanitise = text => text.replace(/[^\w ]/g, '');
```

Comment:
Not at all essential - it's for quick browsing by humans only (e.g. "which `more-ons-theme` is getting the most clicks?"). Plain text will suffice.